### PR TITLE
Cdls-design-process-documentation

### PR DESCRIPTION
**Pull request by Uri-E**
Update the Process tab to include step by step documentation on how new component/new variant requests are handled.

Please review and provide feedback if anything looks out of place.

Geaux Tigers.

### CODE REVIEW

File: docs/03-process.md [hunk 1]
```diff
@@ -3,8 +3,83 @@ title: Design Process
...
+### Introduction
+This document outlines the design process for adding a new component to the UTSA College Design Language System (CDLS). The objective is to seamlessly update an existing component or integrate the new component into the established design system, ensuring consistency and cohesiveness with the existing components.
 
-### More detail to follow
+### Design Process Overview 
+The design process for adding a new component involves the following steps:
+<ol>
+    <li><strong>Design Request</strong>
+        <ul>
+            <li>Stakeholder submits request to the web team through online form (https://provost.utsa.edu/communications/) or by communicating directly with web team through Teams</li>
```
Comment (lalov1) [inline docs/03-process.md:16]: Looks great! I would make "online form", a link, or the provost website a link. To do this in Markdown you would use the following code: 
`[Link Name](Link URL)`

**Review by lalov1 — COMMENTED** [review]
Minor edit suggestion, but not required.

**Review by prd409-utsa — APPROVED** [review]
Looks good.  Maybe link **atomic web design** to Brad Frost's Atomic Design documentation in case someone wants to know more about it...? Most prolly won't though. Just a suggestion and a thought.  [https://bradfrost.com/blog/post/atomic-web-design/](https://bradfrost.com/blog/post/atomic-web-design/)

Otherwise, looks good!

**Review by lalov1 — APPROVED** [review]
Looks good!

**Pull request merged**